zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -3961,6 +3961,23 @@ static int Request_path_watch_init(zhandle_t *zh, int mode,
 /*---------------------------------------------------------------------------*
  * ASYNC API
  *---------------------------------------------------------------------------*/
+#define nonblocking_send(zh, rc) _nonblocking_send(zh, rc, __LINE__)
+static int _nonblocking_send(zhandle_t *zh, int rc, int line)
```

Comment:
this parameter `line ` is not used, why should we declare it ?

Reply:
Thanks for pointing this out.  Formerly this was used because the function called handle_socket_error_msg() on error.  Since this is no longer called, we can omit the macro and the argument.  Updated PR , force-pushed amended version f0539f292